app/views/v1/events/show.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.extract! @event, :id, :name, :date, :periodicity, :owner
```

Comment:
We are not being consistent, as the previous responses had `success: true` in the body. But we can let it as it is, and when we implement the client, we'll see what exactly we need.
I've just commented to keep it in mind :stuck_out_tongue_winking_eye: